### ISSUE TITLE
"Very Hot" alerts should be the only temp alert when thermometer is maxed out

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#101]
The goal of this issue is the try to make sure that we fix the warming alerts as it pertains to when the thermometer is maxed out. 

As part of this issue, I think we should do the following:
- [x] Remove "very Hot" alerts from the current increasing describer, because it is rarely said anyways.
- [x] Create a separate alert system for "very hot" when the thermometer is maxed out.
- [x] Make sure that we don't hear "warmer" or "hotter" alerts when the thermometer is maxed out.

Tagging @terracoda in case she has comments.

**Comment by terracoda** [comment]
@zepumph, this sounds like a good approach.
In my testing this afternoon I only heard "Jiggling more, warmer". I didn't hear any faster-hotter alerts or even-faster-even-hotter alerts. I think it would be good to get at least one hotter-faster type alert in before the max-out alert happens (see my comment in issue #99 (https://github.com/phetsims/friction/issues/99#issuecomment-418806209).

When thermometer stays maxed out through continued friction, I thought alerts like these might work:
- Atoms jiggling {{very fast}}, temperature {{very hot}}.
- Still {{very fast}}, still {{very hot}}.

But a comparative alert might be more true to the model:
- Atoms jiggling {{faster}}, temperature {{hotter}}.
- {{Faster}}, {{hotter}}.

Alert fires as long as all atoms have broken away, thermometer is maxed out, and temperature is mainly increasing. 

When interaction slows, the cooling alerts will kick in.

**Comment by zepumph** [comment]
In the above commits I implemented a few things:
* I accomplished all three of the above checkboxes
* rather than basing things off of the maximum value of the thermometer, I used the evaporation limit. I'm not sure that this should stick around, but it was easier to implement for this first pass.
* I made it so that if temp is greater than the evaporation limit, it will trigger the following alerts in the exact same way (same thresholds/timing) as the "increasing" warmer/hotter/even hotter alerts.

(first) Atoms very fast, temperature very hot.
(subsequent) Jiggling very fast, still very hot.

@terracoda please review.

**Comment by terracoda** [comment]
@zepumph, again as I said in #100, this sounds great. 
I think your intuition was good to use the evaporation point. I think it works well.

Closing this issue :-)
